avsomr/avsomrdrawer.cpp

```diff
@@ -0,0 +1,208 @@
+#include "avsomrdrawer.h"
+
+#include <QPainter>
+
+#include "libmscore/score.h"
+#include "libmscore/measure.h"
+
+#include "avslog.h"
+
+using namespace Ms::Avs;
+
+AvsOmrDrawer::AvsOmrDrawer()
+      {
+
+      }
+
+//---------------------------------------------------------
+//   makeContext
+//---------------------------------------------------------
+
+std::shared_ptr<AvsOmrDrawer::Context> AvsOmrDrawer::makeContext(QPainter* p, const Ms::Score* score) const
+      {
+      IF_ASSERT(p) {
+            return nullptr;
+            }
+
+      IF_ASSERT(score->isMaster()) {
+            return nullptr;
+            }
+
+      const Ms::MasterScore* ms = static_cast<const Ms::MasterScore*>(score);
```

Comment:
Probably better to use `score->masterScore()`

Reply:
fixed